src/main/java/de/craften/plugins/rpgplus/scripting/api/RpgPlusObject.java

```diff
@@ -95,10 +101,18 @@ public void sendMessage(Varargs varargs) {
                 players = LuaTable.listOf(new LuaValue[]{varargs.arg(1)});
             }
             for (int i = 1; i <= players.length(); i++) {
-                Player p = ScriptUtil.getPlayer(players.get(i));
-                for (int j = 2; j <= varargs.narg(); j++) {
-                    p.sendMessage(ChatColor.translateAlternateColorCodes('&', varargs.checkjstring(j)));
+                try {
+                	CommandSender sender = (CommandSender) CoerceLuaToJava.coerce(players.get(i), CommandSender.class);
+                	for (int j = 2; j <= varargs.narg(); j++) {
+                        sender.sendMessage(ChatColor.translateAlternateColorCodes('&', varargs.checkjstring(j)));
+                    }
+                } catch(Exception e) {

```

Comment:
:-1: Please check if `CorceLuaToJava.coerce` returns a `CommandSender` instead of silently catching everything. Exception handling should not be used for _expected_ behavior.